### PR TITLE
Google Workspace Admin Side Changes

### PR DESCRIPTION
**Pull request by kunalbotla**
Admin side changes of Google Workspace

this PR changes

- changes the website we use for the admin DNS visual from dns-lookup.com to nslookup.io
- adds a notice about the plugin
- adds more guidance about domains to approve
- adds code side comments

### CODE REVIEW

File: app/views/admin/google_workspace_process.html.erb [hunk 1]
```diff
@@ -60,6 +60,10 @@
 <% if @g_suite.verifying? %>
   <p>Hack Club Bank is now trying to automatically verify if the user has configured their domain correctly with the TXT record.</p>
   <p>If you would like to check yourself:</p>
+  <ul>
+    <li>Use the HCB Bank Ops Plugin to verify the TXT & MX records and see the current status.</li> <!-- need to publish to Chrome Web Store for internal use - kunal -->
```
Comment (maxwofford) [inline app/views/admin/google_workspace_process.html.erb]:
```suggestion
    <li>Use the HCB Bank Ops Plugin to verify the TXT & MX records and see the current status. Reach out to kunal@hackclub.com or @kunal on slack to get setup with the HCB Bank Ops Plugin.</li>
```

File: app/views/admin/google_workspace_process.html.erb [hunk 2]
```diff
@@ -36,8 +36,8 @@
 <p>Context: <%= link_to "see what the user sees", event_g_suite_overview_path(event_id: @g_suite.event.slug) %>.</p>
 
 <% if @g_suite.creating? %>
-  <p>Is this a valid/reasonable domain name? If so, let's approve it!<br/>This will generate a verification key for this domain and
-  email the organizer to let them know they can start setting up their domain's DNS records.</p>
+  <p>Is this a valid/reasonable domain name? If so, let's approve it!<br/>Check out the URL to see if it seems like they own the domain.<br/>This will generate a verification key for this domain and
+  email the organizer to let them know they can start setting up their domain's DNS records.</p> <!-- maybe add a Hack Club verification token here to verify ownership before approving and creating? - kunal -->
```
Comment (maxwofford) [inline app/views/admin/google_workspace_process.html.erb]:
```suggestion
  email the organizer to let them know they can start setting up their domain's DNS records.</p>
```

How about dropping a required checkbox on the gsuite signup form? No backend needed.

**Review by maxwofford — APPROVED** [review]
Couple suggestions, but overall LGTM. Go ahead and rebase + merge once those are addressed

File: app/views/admin/google_workspace_process.html.erb (hunk 2 above)
Comment (kunalbotla) [reply app/views/admin/google_workspace_process.html.erb]: Do you mean like a checkbox for users's to click to verify ownership?

Comment (maxwofford) [reply app/views/admin/google_workspace_process.html.erb]: Yeah, don't include in this PR, but for the future. I'd like to keep notes like this out of the codebase though unless that feature is going to be implemented in the foreseeable future

Comment (kunalbotla) [reply app/views/admin/google_workspace_process.html.erb]: Sure, I'll take it out for this PR and look into what ways we could implement something for it

**Pull request merged**